packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -119,6 +127,35 @@ export function createTimeSeries(
   };
 }
 
+function transformResource(
+  resource: Resource,
+  projectId: string
+): { type: string; labels: { [key: string]: string } } {
+  const cloudProvider = `${resource.labels[CLOUD_RESOURCE.PROVIDER]}`;
+  // These are the only supported resources
+  if (cloudProvider === 'gcp') {
+    return {
+      type: GCP_GCE_INSTANCE,
+      labels: {
+        instance_id: `${resource.labels[HOST_RESOURCE.ID]}`,
+        project_id: projectId,
+        zone: `${resource.labels[CLOUD_RESOURCE.ZONE]}`,
```

Comment:
As per my experience with OpenCensus Stackdriver exporter, Stackdriver backend will complain expected labels are missing (see: https://github.com/census-instrumentation/opencensus-node/issues/706), which means we need to make sure all labels has valid values and in worst case fallback on global type.

You can very well copy code from OpenCensus: https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-exporter-stackdriver/src/common-utils.ts

Reply:
You can easily reproduce this: 

1. Set type to `gce_instance`
2. Assign valid `instance_id` & `project_id` and keep zone as blank.
3. Try to export data to backend.